test-framework/junit5/src/main/java/io/quarkus/test/junit/GraalVMVersion.java

```diff
@@ -0,0 +1,34 @@
+package io.quarkus.test.junit;
+
+public enum GraalVMVersion {
```

Comment:
Does this really have to be part of the public API now? How come?

Reply:
Good catch, that's left over from some other refactoring I was doing for #24768

Reply:
Gotcha, so that PR is going to need the change but not this one?
If so, probably best to isolate the changes so we don't end up in an intermediate state if something changes in the other PR

Reply:
Turns out that's needed because we pass it as argument to `DisableIfBuiltWithGraalVMOlderThan` in `integration-tests`, e.g.:

```java
    @Test
    @DisableIfBuiltWithGraalVMOlderThan(GraalVMVersion.GRAALVM_22_1)
    public void testTargetWithNestedPost22_1() {
```